lib/tasks/instrumentation_generator/templates/chain.tt

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
```

Comment:
needs the frozen string literal magic comment

Reply:
Resolved in: 94a9deec5307f21a2490f7a4e12e4276e411999b